### PR TITLE
Update documentation (asyncapi support page)

### PR DESCRIPTION
**Pull request by vvenance**

### CODE REVIEW

File: docs/help/specifications-support/asyncapi-support.md [hunk 1]
```diff
@@ -16,29 +12,52 @@ Bump is able to extract:
...
 
-We do not support `X509`, `symmetricEncryption`, `asymmetricEncryption`, `plain`, `scramSha256`, `scramSha512` nor `gssapi.` To describe these authentication types, please use our [custom `x-topics` property](doc-topics.md) for now.
+The following authentication types are **not** supported: 
+- `X509`
+- `symmetricEncryption`
+- `asymmetricEncryption`
+- `plain`
+- `scramSha256`
+- `scramSha512`
+- `gssapi.`
```
Comment (paulRbr) [inline docs/help/specifications-support/asyncapi-support.md:34]:
```suggestion
- `gssapi`
```

It seems there's an old typo in this content :)

File: docs/help/specifications-support/asyncapi-support.md [hunk 2]
```diff
@@ -16,29 +12,52 @@ Bump is able to extract:
...
-Basically, in Async API context, it means that a writeOnly property makes sense in request only (as password for example). Thus, we decided to hide a property defined as writeOnly when it belongs to a subscribe operation (messages produced by the application and sent to the channel).
+- `readOnly` properties are hidden when they belong to a publish operation (messages consumed by the application from the channel).
 
-And a readOnly makes sense in response only (as created_at, updated_at, your call...). Similarly,  we decided to hide a property defined as readOnly when it belongs to a publish operation (messages consumed by the application from the channel).
+:::info
+Not displaying `writeOnly` properties in subscribe operations and `readOnly` properties in publish operations allows the use of same `Schema Object` everywhere it is needed, without generating  confusing informations in the documentation.
+:::
 
-Thus, it becomes easy to use the same Schema Object in different context, for example as seen below:
+Thus, it becomes easy to use the same `Schema Object` in different contexts, for example as seen below:
```
Comment (scharrier) [inline docs/help/specifications-support/asyncapi-support.md:60]: Adding the writeOnly and readOnly behaviors makes sense, but maybe it's too dense here. Do you think we could simplify it?

Also, here are some specific feedbacks:
* typo: `Async API` VS `AsyncAPI`
* using the request/response paradigm is a bit surprising in the context of async apis (as there are no requests/responses but events)
* as the writeOnly/readOnly statement is not related to AsyncAPI specifically, should we start extracting this in a `JSON schema` page, and point to this page from OpenAPI and AsyncAPI ones?

Would you like some help with simplifying the block? We can work together on this :)

File: docs/help/specifications-support/asyncapi-support.md [hunk 3]
```diff
@@ -16,29 +12,52 @@ Bump is able to extract:
 
 ## Partially supported: message object `payload`
 
-Message#payload could be of any type according to the specification, but we only support `SchemaObject` type.
+[Message object](https://github.com/okere-prince/asyncapi/blob/master/versions/2.0.0/asyncapi.md#message-object) field `payload` could be of any type according to the specification, but we only support `SchemaObject` type.
```
Comment (paulRbr) [inline docs/help/specifications-support/asyncapi-support.md:15]:
```suggestion
[Message object](https://www.asyncapi.com/docs/reference/specification/v2.6.0#messageObject) field `payload` could be of any type according to the specification, but we only support `SchemaObject` type.
```

File: docs/help/specifications-support/asyncapi-support.md (hunk 2 above)
Comment (vvenance) [reply docs/help/specifications-support/asyncapi-support.md]: All of the above has been addressed in https://github.com/bump-sh/docs/pull/72/commits/8fc67bc3ed3b722ddd2ced518e927dc57ed31c08. I've extracted the content from openapi and asyncapi support in a JSON schema page and simplified so it makes sense in both OpenAPI and AsyncAPI context. Let me know your thoughts or feedbacks

**Review by scharrier — APPROVED** [review]
LGTM, thanks for the clarification 👍

**Pull request merged**